docs/development-guidelines/development-process.md

```diff
@@ -0,0 +1,40 @@
+## Development Guidelines - Process
+
+For cross-functional teams working on building a template, the following guidelines are recommended.
+
+- Create labels that reflect the different dimensions for features and operations. For example:
+
+| Label Name | Description                          |
+| ---------- | ------------------------------------ |
+| AI         | AI implementation and implementation |
+| Infra AZD        | AZD related         |
+| Infra Bicep| Infra issues related to Bicep bugs or support |
+| Infra Terraform | Infra issues related to Terraform bugs or support |
+| backend    | Application feature backend          |
+| frontend   | Application feature frontend         |
+| Docs       | Repo docs, guidelines, promo videos  |
+| MSI        | Security and auth                    |
```

Comment:
Azure Managed Service Identity (MSI) is now known as Managed Identities for Azure Resources. Having the `MSI` label might be confusing.

Reply:
I will update all labels in repos and this is fixed now in this docs MSI > Managed Identity